traffic_ops/app/lib/API/User.pm

```diff
@@ -277,6 +285,7 @@ sub current {
 			@data, {
 				"id"              => "0",
 				"username"        => $current_username,
+				"tenantId"	  => $self->current_user_tenant(),
```

Comment:
actually, after looking at the code closer maybe it just makes sense to leave it undef for now which is what $self->current_user_tenant() will return in this case. i'm fine with that...